data/hai/unfettered jobs.txt

```diff
@@ -313,8 +313,8 @@ mission "Unfettered Small Medicals Rush"
 	repeat
 	clearance "After speaking with the Unfettered and explaining your presence, they give you permission to land with the much needed medical supplies."
 	name "Express medical stock from <planet stopovers>"
-	description "The Unfettered hospitals on <destination> are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered before <date> for <payment>."
-	deadline 5 1
+	description "The Unfettered hospitals on <destination> are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and swiftly bring it to the Unfettered before <date> for <payment>."
+	deadline 2 1
 	cargo "medical supplies" 15 2 .05
```

Comment:
```suggestion
	cargo "medical supplies" 10 2 .05
```
this would nerf it further, should we want that
I dont want to block it behind positive uhai relations bc it loses interest then - being able to bribe your way there with the risks involved is smth I find interesting